typescript/phoenix-sdk/examples/placeLimitOrder.ts

```diff
@@ -30,7 +36,8 @@ export async function placeLimitOrderExample() {
   const marketAddress = new PublicKey(
     "CS2H8nbAVVEUHWPF5extCSymqheQdkd4d7thik6eet9N"
   );
-  const marketData = phoenixClient.markets.get(marketAddress.toBase58())?.data;
+  const market = phoenixClient.markets.get(marketAddress.toBase58());
+  const marketData = market?.data;
 
   if (marketData === undefined) {
```

Comment:
this should early return if `market === undefined`, then you don't need the ? on line 40